atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -568,6 +569,7 @@ def get_build(self, metadata):
             'start_time': start_time,
             'end_time': int(time.time()),
             'extra': extra,
+            'autorebuild': is_rebuild(self.workflow)
```

Comment:
It definitely can't go here. At the very least it must be inside 'extra'. The question is whether it needs to be in `build.extra.autorebuild` (I think this is best) or `build.extra.image.autorebuild`.